modDesc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" standalone="no" ?>
 <modDesc descVersion="39">
-	<version>5.03.00042</version>
+	<version>5.03.00044</version>
```

Comment:
Should be 43 or 1 plus what ever the current version is

Reply:
Sorry, it kind of was 43 for my mode9 update, but then I decided to make pull request for intersections, so I changed it again. I guess those both pull requests combined into one. 
Still a lot to learn on Github.

Reply:
I do still feel kind of baffled about the whole pull request procedure. 
If there is a way to split my independent changes into separate pull requests so they can be evaluated separately and more easily then I did not found it yet.

Reply:
The easiest way to handle this is to create separate branches in your clone for each feature. Also, if you have many work in progress commits, it is probably easier to squash them before creating the pull request.

Familiarity with git helps a lot, there are many good tutorials on git and the pull request workflow on the net.

Reply:
Thanks for heads up, the only downside is that now I have to read up on "branches" :-(. Oh well it would be useful I am sure.

I am not in particular hurry to commit as many changes to CP as soon as possible (it works on my FS17...) , so limiting it to one change at a time may be the best way forward.

Shovel change is trivial and standalone. It's so simple that most gamers can do it themselves. Or just buy larger/moded tele handler and showel and so sidestep the issue completely. May be not even needs a merge.

Crossings are standalone, but not very useful in itself on the grand scheme of things - what use is even proper waypoint merge if tractors would tip over every time they have to make a turn as nothing else instructs them to slow down?

Recording waypoints with slow vehicles and recording a slowdown before each and every intersection because there may need to be a direction change is doable, but it is so ... much ... pain for the player. That's not even counting even worse pain of watching tens of vehicles crawl around from one intersection to next where they do not actually need to make any turns rather than zipping right past them at their individual full speed.

I do intend to finish crossings pull request all proper - as far and as soon as my increasing but still tiny experience with Github allows.

Waypoint filtering part (no pull request yet) helps a lot with all the pain - which is why I made it for my world. But let's face it - it is an ugly hack at this point by any decent programmers definition - and I am not really sure I know an elegant way to root it out of CourseManagement's loadCourse and place it somewhere into the drive.lua.

Should I even try to make a pull request on it before I know that way? That might take a long while.

Reply:
Not sure what you mean by waypoint filtering but don't change anything in drive.lua, that file is due for a major overhaul.